aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -612,6 +614,39 @@ mod loader {
             self
         }
 
+        /// Override the [`StoppedStreamProtectionConfig`] used to build [`SdkConfig`](aws_types::SdkConfig).
+        ///
+        /// This configures stalled stream protection. When enabled, download streams
+        /// that stop (stream no data) for longer than a configured grace period will return an error.
+        ///
+        /// By default, streams that transmit less than one byte per-second for five seconds will
+        /// be cancelled.
+        ///
+        /// _Note_: When an override is provided, the default implementation is replaced.
+        ///
+        /// # Examples
+        /// ```no_run
+        /// # async fn create_config() {
+        /// use aws_smithy_types::stalled_stream_protection::StalledStreamProtectionConfig;
```

Comment:
we shoudl re-export it and have the example use the re-exported struct to encourage customers to do the same

Reply:
I already reexport it for service crates. I'll add a reexport for aws-config too.